web/src/main/webapp/components/commonDirectives/rhsPatientAgTable/rhsPatientAgTable.directive.js

```diff
@@ -58,6 +62,10 @@ angular.module('rehabstodApp')
               });
             }
           });
+
+          $scope.isExpiredCertificate = function(enddate){
+            return Date.parse(enddate) < Date.parse($scope.todaysDate);
```

Comment:
Wondering if this todays date handling could be a bit more efficient and simpler.

Now the code (above) creates current date without time by creating a date and "manually" creating a formatted string. 

Could it not just create var todaysDate = new Date() and then todaysDate.setHours(0,0,0,0) (see following: https://stackoverflow.com/questions/2698725/comparing-date-part-only-without-comparing-time-in-javascript)

And in this isExpiredCertificate function just do $scope.today.getTime() (see following: https://www.w3schools.com/jsref/jsref_gettime.asp)

Please look at it and if you can simplify then do... if you have reasons that I'm missing please keep current implementation.

Reply:
Thank you for the suggestion. We have used it and tested, and it seems to work fine.